Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,26 @@
--  user wants to see the info page 
--  user wants to apply as artist 
-- user wants to donate 
--  user wants to have a prostictic worked on
\ No newline at end of file
+Actors: Visitor, Artist, Admin
+
+Visitor - View info page, View Gallery, Donate, RequestProstetic, Liking a Picture.
```

Comment:
The format of describing cases should be changed.
The first sentence is good. ("liking a picture" should be "like a picture")

Reply:
Each case should be described.
Bold the case name and follow this format:
**Case name**- description.